LoopKit/InsulinKit/CachedInsulinDeliveryObject+CoreDataProperties.swift

```diff
@@ -26,6 +26,7 @@ extension CachedInsulinDeliveryObject {
     @NSManaged public var primitiveScheduledBasalRate: NSNumber?
     @NSManaged public var primitiveProgrammedTempBasalRate: NSNumber?
     @NSManaged public var primitiveReason: NSNumber?
-    @NSManaged public var createdAt: Date
-
+    @NSManaged public var createdAt: Date?
```

Comment:
Hmm, good catch; I think it should not be optional; this might have a been a merge issue. I think it was optional at one point in the Tidepool branch as Darin was updating the model to track creation time separately from start time. It does appear to compile as a non-optional, and is non-optional in the data model, so I'll change this.